spinalcordtoolbox/scripts/sct_merge_images.py

```diff
@@ -97,7 +97,7 @@ def get_parser():
     misc = parser.add_argument_group('MISC')
     misc.add_argument(
         "-r",
-        type=bool,
+        type=int,
         help='Remove temporary files.',
         required=False,
         default=Param().rm_tmp,
```

Comment:
This one was sort of weird!

The default value for `-r` (`Param().rm_tmp`) was `True`, yet its `choices=(0, 1)`, so I'm not sure how this one ever actually... worked? Shouldn't a boolean value not be a valid choice?

Still, its associated check is:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/43a50029187b4a9512cb58149210d45ad3f3227f/spinalcordtoolbox/scripts/sct_merge_images.py#L194-L196

So, there should be no functional change going from `bool` to `int`.